### PR TITLE
bloodsucker fixes

### PR DESCRIPTION
**Pull request by tattax**
# Document the changes in your pull request

fixes the station arrival of bloodsucker + minor things that were reported and clarifying some stuff

also ports https://github.com/fulpstation/fulpstation/pull/542 which i didn't port fully for some reason;

https://github.com/yogstation13/Yogstation/issues/14111
https://github.com/yogstation13/Yogstation/issues/14082
https://github.com/yogstation13/Yogstation/issues/14108
https://github.com/yogstation13/Yogstation/issues/13989
https://github.com/yogstation13/Yogstation/issues/14132

:cl: JohnFulpWillard, Tatax, ToastyBiome
bugfix: bloodsucker bugfixes
/:cl:

### CODE REVIEW

File: code/modules/antagonists/bloodsuckers/bloodsucker_frenzy.dm [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 /obj/screen/alert/status_effect/frenzy
 	name = "Frenzy"
-	desc = "You are in a Frenzy! You are entirely Feral and, depending on your Clan, fighting for your life!"
+	desc = "You are in a Frenzy! You are entirely Feral and, depending on your Clan, fighting for your life! <i>Feeding</i> while in this state is instant!"
```
Comment (ToasterBiome) [inline code/modules/antagonists/bloodsuckers/bloodsucker_frenzy.dm]:
```suggestion
	desc = "You are in a Frenzy! You are entirely feral, and depending on your Clan - fighting for your life! <i>Feeding</i> while in this state is instant!"
```

File: code/modules/antagonists/bloodsuckers/structures/bloodsucker_crypt.dm [hunk 2]
```diff
@@ -1005,9 +1005,9 @@
 	can_buckle = TRUE
 	anchored = FALSE
 	Ghost_desc = "This is a magical candle which drains at the sanity of non Bloodsuckers and Vassals.\n\
-		Vassals can turn the candle on manually, while Bloodsuckers can do it from a distance."
+		Vassals can also turn the candle on, which slowly drains anyone near it."
```
Comment (ToasterBiome) [inline code/modules/antagonists/bloodsuckers/structures/bloodsucker_crypt.dm]: Drains what?

File: code/modules/antagonists/monsterhunter/monstertrack.dm [hunk 3]
```diff
@@ -6,7 +6,7 @@
 	icon_icon = 'icons/mob/actions/actions_bloodsucker.dmi'
 	background_icon_state = "vamp_power_off"
 	button_icon_state = "power_hunter"
-	power_flags = NONE
+	power_flags = BP_AM_TOGGLE
```
Comment (ToasterBiome) [inline code/modules/antagonists/monsterhunter/monstertrack.dm]: Why is this ability still a toggle?

File: code/modules/events/sinfuldemon.dm [hunk 4]
```diff
@@ -40,8 +40,16 @@
 	message_admins("[ADMIN_LOOKUPFLW(sinfuldemon)] has been made into a demon of sin by an event.")
 	log_game("[key_name(sinfuldemon)] was spawned as a demon of sin by an event.")
 	var/datum/job/jobdatum = SSjob.GetJob("Assistant")
-	sinfuldemon.job = jobdatum.title
+	if(SSshuttle.arrivals)
```
Comment (ToasterBiome) [inline code/modules/events/sinfuldemon.dm]: should be in it's own PR

**Review by ToasterBiome — DISMISSED** [review]

File: code/modules/antagonists/bloodsuckers/structures/bloodsucker_crypt.dm (hunk 2 above)
Comment (tattax) [reply code/modules/antagonists/bloodsuckers/structures/bloodsucker_crypt.dm]: drains gang

File: code/modules/events/sinfuldemon.dm (hunk 4 above)
Comment (tattax) [reply code/modules/events/sinfuldemon.dm]: :/

File: code/modules/antagonists/monsterhunter/monstertrack.dm (hunk 3 above)
Comment (tattax) [reply code/modules/antagonists/monsterhunter/monstertrack.dm]: toggle actually works better than whatever it was doing before with NONE

File: code/modules/antagonists/bloodsuckers/bloodsucker_frenzy.dm (hunk 1 above)
Comment (tattax) [reply code/modules/antagonists/bloodsuckers/bloodsucker_frenzy.dm]: image

Comment (ToasterBiome) [reply code/modules/antagonists/bloodsuckers/bloodsucker_frenzy.dm]: https://www.grammarly.com/blog/capitalization-rules/

File: code/modules/antagonists/monsterhunter/monstertrack.dm (hunk 3 above)
Comment (ToasterBiome) [reply code/modules/antagonists/monsterhunter/monstertrack.dm]: Ok but why is it still a toggle

File: code/modules/antagonists/bloodsuckers/bloodsucker_frenzy.dm (hunk 1 above)
Comment (tattax) [reply code/modules/antagonists/bloodsuckers/bloodsucker_frenzy.dm]: ip catcher

File: code/modules/antagonists/monsterhunter/monstertrack.dm (hunk 3 above)
Comment (tattax) [reply code/modules/antagonists/monsterhunter/monstertrack.dm]: toggle auto updates the power when it is deactivated, with no flags it actually just keeps the background unupdated

**Review by JamieD1 — APPROVED** [review]
